src/Extractor/ContentExtractor.php

```diff
@@ -669,4 +686,51 @@ private function extractBody($detectBody, $xpathExpression, \DOMNode $node, $typ
 
         return false;
     }
+
+    /**
+     * Extract and apply a callback to an entity according to a pattern
+     *
+     * @param string $entity            Entity to look for
```

Comment:
I'm ok with the use of `$this->{$entity}` but please put some example for `$entity` in the phpDoc. This will be less obscure